### PR TITLE
feat(dropdowns): added dropdown, context selector full height mods

### LINKED ISSUE #4105: Update masthead context selector / dropdown border colors
Masthead context selector / dropdown border color spec

Screen Shot 2021-05-26 at 10 57 07 AM

### LINKED ISSUE #4109: feat(dropdowns): add full height context selector, dropdown styling mods

### PR DESCRIPTION
**Pull request by mattnolting**
closes #4109

### CODE REVIEW

File: src/patternfly/demos/Masthead/examples/Masthead.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        brand--img-url--base='/assets/images/logo__pf--reverse--base.png'
+        brand--img-url='/assets/images/logo__pf--reverse--base.svg'
+        brand--img-url-on-md='/assets/images/logo__pf--reverse-on-md.svg'}}
+    {{/masthead-brand}}
+  {{/masthead-main}}
+  {{#> masthead-content}}
+    {{#> toolbar}}
+      {{#> toolbar-content-section}}
+        {{#> toolbar-item}}
+          {{#> context-selector context-selector--id="context-selector-collapsed-example" context-selector--label-text="Selected project" context-selector--IsExpanded="true"}}
```
Comment (mcoker) [inline src/patternfly/demos/Masthead/examples/Masthead.md:27]: Do you want to add the full-height mods here?

File: src/patternfly/demos/Masthead/examples/Masthead.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+                  {{/button}}
+                {{/input-group}}
+              {{/context-selector-menu-search}}
+              {{> __context-selector-menu-menu}}
+            {{/context-selector-menu}}
+          {{/context-selector}}
+        {{/toolbar-item}}
+        {{#> toolbar-group toolbar-group--modifier="pf-m-align-right"}}
+          {{#> toolbar-item toolbar-item--modifier="pf-m-hidden pf-m-visible-on-lg"}}
+            {{> dropdown dropdown--id="dropdown-expanded" dropdown--IsExpanded="true" dropdown-toggle--text="Expanded dropdown"}}
```
Comment (mcoker) [inline src/patternfly/demos/Masthead/examples/Masthead.md:51]: and add the full-height mod here?

**Comment by mattnolting** [comment]
@mcoker updated

**Review by mcoker — COMMENTED** [review]
Looks good! Couple of questions about the docs.

Also these menus are running off screen. Maybe make them not expanded and we can make them expanded in a full page demo with the new masthead? Also the top borders make it look a little weird. Is that something we'll fix in the masthead component?

**Review by mcoker — APPROVED** [review]
L🤩TM! Want to create an issue to add this to the menu toggle? The menu toggle also has a better way of defining the bottom border so this angle doesn't appear

Screen Shot 2021-06-15 at 11 01 59 AM

**Pull request merged**